ui/packages/catalog-ui-search/src/main/webapp/component/search-form-list/search-form-list.view.js

```diff
@@ -134,9 +134,10 @@ module.exports = Marionette.ItemView.extend({
     })
     if (currentQuery.get('type') === 'custom') {
```

Comment:
❓ the `type` was just set to `custom` is there really need for the following conditional block/will this ever fall through to the `else` clause here?

Reply:
good point